.env.example

```diff
@@ -18,9 +18,7 @@ INFLUXDB_HTTP_AUTH_ENABLED=false
 
 NODE_PORT=8000
 NODE_ENV=development
+ACCELERATION_TOPIC=Thingy Motion Service/Thingy Motion Raw data
 
-MAC_THINGY1=e8:e0:72:6b:92:a3
```

Comment:
@pf15ese I don't think this has to be removed

Reply:
I think it should; The machine now has a sensorIdentifier field which includes this address. But there are still uses, i will clean them up